cd/publish_images.sh

```diff
@@ -58,7 +58,9 @@ docker login -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}"
 pushTags gchq/hdfs "${HADOOP_VERSION}" "${APP_VERSION}"
 pushTags gchq/accumulo "${ACCUMULO_VERSION}" "${APP_VERSION}"
 pushTags gchq/gaffer "${GAFFER_VERSION}" "${APP_VERSION}"
+pushTags gchq/gaffer "${GAFFER_VERSION}-accumulo-${ACCUMULO_VERSION}" "${APP_VERSION}"
 pushTags gchq/gaffer-rest "${GAFFER_VERSION}" "${APP_VERSION}"
+pushTags gchq/gaffer-rest "${GAFFER_VERSION}-accumulo-${ACCUMULO_VERSION}" "${APP_VERSION}"
```

Comment:
Have we confirmed that this does what we expect? Looking at the `pushTags` function above, I'm not sure it will work. It expects there to be an image with both tags locally already. I think an alternative version of that function could be created to do this.

Reply:
I will have a look and test locally, good point though

Reply:
Erroneous change has been removed, `publish_images` should be updated in a future PR